src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectAutoCRContainer.tsx

```diff
@@ -0,0 +1,141 @@
+  const toast = useToast();
+  const history = useHistory();
+
+  const [projectManagerId, setProjectManagerId] = useState<string | undefined>();
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>();
+
+  const { mutateAsync, isLoading } = useCreateStandardChangeRequest();
+  const {
+    data: allLinkTypes,
+    isLoading: allLinkTypesIsLoading,
+    isError: allLinkTypesIsError,
+    error: allLinkTypesError
+  } = useAllLinkTypes();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (!allLinkTypes || allLinkTypesIsLoading) return <LoadingIndicator />;
+  if (allLinkTypesIsError) return <ErrorPage message={allLinkTypesError.message} />;
+
+  const requiredLinkTypeNames = getRequiredLinkTypeNames(allLinkTypes);
+
+  const defaultValues = {
+    name: String(),
+    budget: 0,
+    summary: String(),
+    teamId: String(),
+    carNumber: 0,
+    links: [],
+    crId: '',
+    goals: [],
+    features: [],
+    constraints: [],
+    rules: [],
+    projectLeadId,
+    projectManagerId
+  };
+
+  const schema = yup.object().shape({
+    name: yup.string().required('Name is required!'),
+    carNumber: yup.number().min(0).required('A car number is required!'),
+    teamId: yup.string().required('A Team Id is required'),
+    budget: yup.number().optional(),
+    summary: yup.string().required('Summary is required!'),
+    projectLeadId: yup.number().optional(),
+    projectManagerId: yup.number().optional(),
+    links: yup
+      .array()
+      .optional()
+      .of(
+        yup
+          .object()
+          .optional()
+          .shape({
+            linkTypeName: yup.string().optional(),
+            url: yup.string().optional().url('Invalid URL')
+          })
+      )
+  });
+
+  const onSubmit = async (data: ProjectFormInput) => {
+    const { name, budget, summary, links, teamId, carNumber } = data;
+
+    const rules = data.rules.map((rule) => rule.detail);
+    const goals = mapBulletsToStrings(data.goals);
+    const features = mapBulletsToStrings(data.features);
+    const otherConstraints = mapBulletsToStrings(data.constraints);
+
+    try {
+      const projectPayload: ProjectProposedChangesCreateArgs = {
+        name,
+        summary,
+        status: WbsElementStatus.Active,
+        teamIds: [teamId],
+        budget,
+        rules,
+        newProject: true,
+        goals,
+        features,
+        otherConstraints,
+        links,
+        projectLeadId: projectLeadId ? parseInt(projectLeadId) : undefined,
+        projectManagerId: projectManagerId ? parseInt(projectManagerId) : undefined,
+        carNumber: carNumber
+      };
+      const payload: CreateStandardChangeRequestPayload = {
+        wbsNum: {
+          carNumber: carNumber,
+          projectNumber: 0,
+          workPackageNumber: 0
+        },
+        type: ChangeRequestType.Issue,
+        what: name,
+        why: [{ explain: 'explanation', type: ChangeRequestReason.Competition }],
```

Comment:
why are these being hardcoded??

Reply:
if you are you need to make it sopmething like “new Project for <project name”

Reply:
the type should not be for competition

Reply:
What should the type be? I was just hardcoding those because I didn't know how else to get them.

Reply:
Let's go with initialization